### ISSUE TITLE
The SSL certificate has expired

### ISSUE THREAD

**Issue by PantelemonVictorStefan** [eclipse-ee4j/glassfish#22760]
I don't know if you are aware but the SSL certificate has expired. This causes logger to crash or something and useful stuff you may want to see in console such as exception stack trace and prints won't show up. This problem is already 1 year old but for some reason nobody has fixed it. 

This is the console output:

2019-06-16T02:52:05.711+0300|Info: Running GlassFish Version: GlassFish Server Open Source Edition  5.0  (build 25)
2019-06-16T02:52:05.713+0300|Info: Server log file is using Formatter class: com.sun.enterprise.server.logging.ODLLogFormatter
2019-06-16T02:52:05.812+0300|Info: Realm [admin-realm] of classtype [com.sun.enterprise.security.auth.realm.file.FileRealm] successfully created.
2019-06-16T02:52:05.815+0300|Info: Realm [file] of classtype [com.sun.enterprise.security.auth.realm.file.FileRealm] successfully created.
2019-06-16T02:52:05.819+0300|Info: Realm [certificate] of classtype [com.sun.enterprise.security.auth.realm.certificate.CertificateRealm] successfully created.
2019-06-16T02:52:06.151+0300|Info: Authorization Service has successfully initialized.
2019-06-16T02:52:06.182+0300|Info: HV000001: Hibernate Validator 6.0.2.Final
2019-06-16T02:52:06.224+0300|Info: Registered org.glassfish.ha.store.adapter.cache.ShoalBackingStoreProxy for persistence-type = replicated in BackingStoreFactoryRegistry
2019-06-16T02:52:06.260+0300|Info: Grizzly Framework 2.4.1 started in: 22ms - bound to [/0.0.0.0:8080]
2019-06-16T02:52:06.283+0300|Info: Grizzly Framework 2.4.1 started in: 4ms - bound to [/0.0.0.0:8181]
2019-06-16T02:52:06.291+0300|Info: Grizzly Framework 2.4.1 started in: 3ms - bound to [/0.0.0.0:4848]
2019-06-16T02:52:06.324+0300|Info: Grizzly Framework 2.4.1 started in: 1ms - bound to [/0.0.0.0:3700]
2019-06-16T02:52:06.605+0300|Info: GlassFish Server Open Source Edition  5.0  (25) startup time : Felix (13,771ms), startup services(1,190ms), total(14,961ms)
2019-06-16T02:52:06.707+0300|Info: JTS5014: Recoverable JTS instance, serverId = [100]
2019-06-16T02:52:07.111+0300|Info: JMXStartupService has started JMXConnector on JMXService URL service:jmx:rmi://192.168.0.3:8686/jndi/rmi://192.168.0.3:8686/jmxrmi
2019-06-16T02:52:07.170+0300|Info: Grizzly Framework 2.4.1 started in: 6ms - bound to [/0.0.0.0:7676]
2019-06-16T02:52:07.173+0300|Info: Registered com.sun.enterprise.glassfish.bootstrap.osgi.EmbeddedOSGiGlassFishImpl@3a401749 as OSGi service registration: org.apache.felix.framework.ServiceRegistrationImpl@75cf0de5.
2019-06-16T02:52:10.492+0300|Info: visiting unvisited references
2019-06-16T02:52:11.044+0300|Severe: The SSL certificate has expired: [
[
  Version: V3
  Subject: OU=Equifax Secure Certificate Authority, O=Equifax, C=US
  Signature Algorithm: SHA1withRSA, OID = 1.2.840.113549.1.1.5
  Key:  Sun RSA public key, 1024 bits
  modulus: 135786214035069526348186531221551781468391756233528066061569654028671100866720352830303278016129003918213826297308054231261658522889438712013757624116391437358730449661353175673177742307421061340003741057138887918110217006515773038453829253517076741780039735595086881329494037450587568122088113584549069375417
  public exponent: 65537
  Validity: [From: Sat Aug 22 19:41:51 EEST 1998,
               To: Wed Aug 22 19:41:51 EEST 2018]
  Issuer: OU=Equifax Secure Certificate Authority, O=Equifax, C=US
  SerialNumber: [    35def4cf]
Certificate Extensions: 7
[1]: ObjectId: 1.2.840.113533.7.65.0 Criticality=false
Extension unknown: DER encoded OCTET string =
0000: 04 0D 30 0B 1B 05 56 33   2E 30 63 03 02 06 C0     ..0...V3.0c....
[2]: ObjectId: 2.5.29.35 Criticality=false
AuthorityKeyIdentifier [
KeyIdentifier [
0000: 48 E6 68 F9 2B D2 B2 95   D7 47 D8 23 20 10 4F 33  H.h.+....G.# .O3
0010: 98 90 9F D4                                        ....
]
]
[3]: ObjectId: 2.5.29.19 Criticality=false
BasicConstraints:[
  CA:true
  PathLen:2147483647
]
[4]: ObjectId: 2.5.29.31 Criticality=false
CRLDistributionPoints [
  [DistributionPoint:
     [CN=CRL1, OU=Equifax Secure Certificate Authority, O=Equifax, C=US]

**Comment by bshannon** [comment]
I can't reproduce this problem, but I seem to be running a different build.
Where exactly did you get the build of GlassFish that you're running?
Can you reproduce this with the official 5.1 release from [here](https://eclipse-ee4j.github.io/glassfish/download)?

**Comment by PantelemonVictorStefan** [comment]
I got the build from https://javaee.github.io/glassfish/download
It is the first result I get when I google "glassfish download".
I tested the official 5.1 release. Works fine. Solves my problem, thank you very much!

A workaround for the 5.0 build was removing all of the certificates from the .jsk files. Yet I do not know why the expired certificate prevented the console from displaying messages.
